File: Deduction/Proposition/Abstraction/Parenthesis.cs
using System.Collections.Generic;

namespace Deduction.Proposition.Abstraction
{
    public class Parenthesis : Connective
    {
        public Parenthesis(params IMember[] parameters)
            : base(parameters)
        {
        }

        public Parenthesis(IEnumerable<IMember> parameters)
            : base(parameters)
        {
        }

        public override int ParameterCount
        {
            get
            {
                return 0;
            }
        }
        public override bool RightAssociative
        {
            get
            {
                return false;
            }
        }

        public override bool Operation(bool[] values)
        {
            return false;
        }
    }
}

File: DeductionJava/src/com/eser/ozvataf/deduction/Proposition/Abstraction/Parenthesis.java
package com.eser.ozvataf.deduction.Proposition.Abstraction;

/**
 * Created with IntelliJ IDEA.
 * User: larukedi
 * Date: 13/01/14
 * Time: 05:43
 * To change this template use File | Settings | File Templates.
 */
public class Parenthesis extends Connective {
    public Parenthesis(IMember... parameters) {
        super(parameters);
    }

    @Override
    public int getParameterCount() {
        return 0;
    }

    @Override
    public boolean operation(boolean[] values) {
        return false;
    }
}
